File: HTML/WebContent/JS/outjs/array.js
/**
 * 배열
 */
/*
var a=[10,20,30];
var sum=0;
for(var i=0;i<3;i++){
	sum+=a[i];  //sum=sum+a[i];
}
alert(sum);

length
*/
//아래 데이터는 A학교의 학생들의 국어점수 성적입니다.
/*	a : 52
	b : 73
	c : 100
	d : 32
	e : 57
	f : 100
	g : 31
	h : 20
위 학생의 총점과 평균을 구하세요.
*/
	
var a=[52,73,100,32,57,100,31,20];
var sum =0;
for(var i=0;i<a.length;i++){
	sum+=a[i];
}
alert("총점 : "+sum);

alert("평균 : "+ sum/a.length);





var b=[50,70,100,33,59,100,33,29];
var sum =0;
for(var i=0;i<b.length;i++){
	sum+=b[i];
}
alert("총점 : "+sum);

alert("평균 : "+ sum/b.length);




var c=[72,53,89,84,15,90,56,75];
var sum =0;
for(var i=0;i<c.length;i++){
	sum+=c[i];
}
alert("총점 : "+sum);

alert("평균 : "+ sum/c.length);

File: JAVAEXAMPLE/src/array/Array.java
package array;

import java.util.Scanner;

public class Array {

	public static void main(String[] args) {
		// 배열 선언 및 초기화
		// 10 명의 학생의 점수에 총점, 평균, 최대값, 최소값을 구해보자
		//79, 88, 91, 33, 99, 55, 95, 60, 72, 57
		
		
		Scanner s = new Scanner(System.in);		
		int[] score = new int[10];
		int sum=0;
		float avg=0f;
		int max=0;
		int min=100;
		for (int i=0;i<score.length;i++) {
			System.out.println((i+1)+"번째 학생의 점수를 입력하시오.");
			
			score[i]=s.nextInt();
			sum+=score[i];
			avg=sum/(float)score.length; // 실수끼리 나누었을때는 소수점이 표현 x. 표현 안된 상태로 avg따라 f가 되면 소용x. 그래서 나눠주는 실수 한군데에 f로 바꿔줘야함!
		
			
			if (score[i]>max) {
				max=score[i];
			}
			if (score[i]<min) {
				min=score[i];
			}
		}
		System.out.println("총점 : "+sum+"점");
		System.out.println("평균 : "+avg+"점");
		System.out.println("최고점수 : "+max+"점");
		System.out.println("최저점수 : "+min+"점");
		
		
		
	}

}
